src/components/core/header.js

```diff
@@ -157,39 +169,84 @@ const Warning = () => (
   </PopperTooltip>
 );
 
+const firstItemStyle = {
+  marginBottom: "0",
+  borderTopLeftRadius: "5px",
+  borderTopRightRadius: "5px"
+};
+
+const lastItemStyle = {
+  marginBottom: "0",
+  borderBottomLeftRadius: "5px",
+  borderBottomRightRadius: "5px"
+};
+
 const Header = () => {
   const categories = useAllCategoriesMeta();
+  const [showDropdown, setShowDropdown] = useState(false);
+  const [selectedItem, setSelectedItem] = useState();
+
+  const openModal = () => setShowDropdown(true);
+  const closeModal = () => setShowDropdown(false);
+  const setVersion = version => {
+    closeModal();
+    setSelectedItem(version);
+    navigate(`${window.location.pathname}/${version}`);
+  };
 
   return (
     <HeaderWrapper id="header">
       <LogoWrapper>
         <a href="/">
           <LiveChatLogo
-            css={css`
-              margin: 0 10px -6px 0;
-              display: block;
-            `}
+            style={{ margin: "0 10px -6px 0", display: "block" }}
             width={80}
           />
         </a>
-        <Link
-          to="/"
-          css={css`
-            color: white !important;
-            text-decoration: none;
-            font-weight: 600;
-            white-space: nowrap;
-            &:hover {
-              text-decoration: none;
-            }
-          `}
-        >
-          Platform Docs
-        </Link>
+        <StyledLink to="/">Platform Docs</StyledLink>
       </LogoWrapper>
+
       <MenuListWrapper>
         <MenuList>
           <Warning />
+          <div style={{ margin: "0 20px" }}>
+            <Dropdown
+              isVisible={showDropdown}
+              onClose={closeModal}
+              triggerRenderer={({ ref }) => (
+                <Button onClick={openModal} ref={ref}>
+                  {selectedItem || "Version"}
+                  <svg
+                    width="24px"
+                    height="24px"
+                    viewBox="0 0 24 24"
+                    fill="#424d57"
+                    style={{ marginLeft: "5px", marginRight: "-10px" }}
+                  >
+                    <path d="M7 10l5 5 5-5H7z"></path>
+                  </svg>
+                </Button>
+              )}
+            >
+              <DropdownList
+                items={[
+                  {
+                    itemId: 0,
+                    content: "v.3.1",
+                    onItemSelect: () => setVersion("v.3.1"),
+                    style: firstItemStyle
+                  },
+                  {
+                    itemId: 1,
+                    content: "v.3.2",
```

Comment:
I thought we are going to have most recent version to be marked as `current` or something similar ?